### PR TITLE
Refactor benchmark

### PR DESCRIPTION
**Pull request by fengyuentau**
Changes:

- [x] use mean as default for benchmark results.
- [x] change result representation to something like this:
   ```shell
    $ python benchmark.py --cfg ./config/face_detection_yunet.yaml
    mean=1.17, median=1.19, min=1.06, input size=[160, 120], model: YuNet with ['face_detection_yunet_2022mar.onnx']
    mean=8.66, median=8.96, min=8.17, input size=[640, 480], model: YuNet with ['face_detection_yunet_2022mar.onnx']
    mean=1.37, median=1.48, min=1.06, input size=[160, 120], model: YuNet with ['face_detection_yunet_2022mar-act_int8-wt_int8-quantized.onnx']
    mean=10.96, median=11.17, min=8.17, input size=[640, 480], model: YuNet with ['face_detection_yunet_2022mar-act_int8-wt_int8-quantized.onnx']
   ```
- [x] add flag `--all` and `--cfg_exclude` to run all benchmarks in one command with exclusions:
   ```shell
    $ python benchmark.py --all --model_exclude license_plate_detection_lpd_yunet_2023mar_int8.onnx:human_segmentation_pphumanseg_2023mar_int8.onnx
    Benchmarking ...
    backend=cv.dnn.DNN_BACKEND_OPENCV
    target=cv.dnn.DNN_TARGET_CPU
    mean       median     min        input size   model
    0.58       0.67       0.48       [160, 120]   YuNet with ['face_detection_yunet_2022mar.onnx']
    0.82       0.81       0.48       [160, 120]   YuNet with ['face_detection_yunet_2022mar_int8.onnx']
    6.18       6.33       5.83       [150, 150]   SFace with ['face_recognition_sface_2021dec.onnx']
    7.42       7.42       5.83       [150, 150]   SFace with ['face_recognition_sface_2021dec_int8.onnx']
    3.32       3.46       2.76       [112, 112]   FacialExpressionRecog with ['facial_expression_recognition_mobilefacenet_2022july.onnx']
    4.27       4.22       2.76       [112, 112]   FacialExpressionRecog with ['facial_expression_recognition_mobilefacenet_2022july_int8.onnx']
    4.68       5.04       4.36       [224, 224]   MPHandPose with ['handpose_estimation_mediapipe_2023feb.onnx']
   ```
- [x] update benchmark results which uses mean as default.

### CODE REVIEW

**Comment by fengyuentau** [comment]
When trying to run benchmarks on all models, I found the quantized PP-HumanSeg and quantized LPD-YuNet are not supported by OpenCV DNN:

```
[ERROR:0@11.647] global onnx_importer.cpp:1054 handleNode DNN/ONNX: ERROR during processing node with 5 inputs and 1 outputs: [QLinearSoftmax]:(onnx_node!p2o.Softmax.0_quant) from domain='com.microsoft'
Traceback (most recent call last):
  File "opencv_zoo/benchmark/benchmark.py", line 191, in <module>
    model = model_handler(*model_path, **model_config)
  File "opencv_zoo/models/human_segmentation_pphumanseg/pphumanseg.py", line 16, in __init__
    self._model = cv.dnn.readNet(self._modelPath)
cv2.error: OpenCV(4.7.0) /Users/xperience/GHA-OCV-Python/_work/opencv-python/opencv-python/opencv/modules/dnn/src/onnx/onnx_importer.cpp:1073: error: (-2:Unspecified error) in function 'handleNode'
> Node [QLinearSoftmax@com.microsoft]:(onnx_node!p2o.Softmax.0_quant) parse error: OpenCV(4.7.0) /Users/xperience/GHA-OCV-Python/_work/opencv-python/opencv-python/opencv/modules/dnn/src/net_impl.hpp:108: error: (-2:Unspecified error) Can't create layer "onnx_node!p2o.Softmax.0_quant" of type "com.microsoft.QLinearSoftmax" in function 'getLayerInstance'
> 
```

Looks like operator `QLinearSoftmax` is not yet supported. Will create an issue and pull request for this.

**Comment by fengyuentau** [comment]
Benchmark results on new hardware will be added in other PR.

**Review by zihaomu — APPROVED** [review]
LGTM

File: benchmark/utils/metrics/tracking.py [hunk 1]
```diff
@@ -8,8 +8,8 @@ class Tracking(BaseMetric):
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
 
-        if self._warmup or self._repeat:
-            print('warmup and repeat in metric for tracking do not function.')
+        # if self._warmup or self._repeat:
```
Comment (zihaomu) [inline benchmark/utils/metrics/tracking.py:11]: remove?

**Review by WanliZhong — APPROVED** [review]
LGTM！

**Comment by fengyuentau** [comment]
👍

**Pull request merged**